config/jobs/kubernetes/sig-node/crio.yaml

```diff
@@ -26,8 +26,10 @@ periodics:
       env:
       - name: GOPATH
         value: /go
+      - name: IGNITION_INJECT_GCE_SSH_PUBLIC_KEY_FILE
+        value: "1"
   annotations:
-    testgrid-dashboards: sig-node-cri-o, sig-release-master-informing
+    testgrid-dashboards: sig-node-cri-o, sig-node-release-blocking
```

Comment:
For which reason do we remove it from `sig-release-master-informing`?

Reply:
We want to move that job from `informing` to `blocking`. Is there any formal process to make that transition? In our last sig node CI meeting we had decided that this job is a good candidate to include in `sig-node-release-blocking` tab.

Reply:
@harche yes, but is the removal from the SIG Release dashboard intentional?

Reply:
@harche we’re speaking about 2 different dashboards: SIG Node vs SIG Release 

While the job is already on SIG Release master informing, I’m assuming your goal is to promote it to SIG Node release blocking. Removing it from the SIG Release dashboard means that the release team will no longer track the job, which is probably not what we want.